sdk/samples/keyvault/keyvault/samples/src/keys_client_example.c

```diff
@@ -40,11 +40,11 @@
  *
  */
 
+#include <az_keyvault.h>
```

Comment:
Even though this is going away (in https://github.com/Azure/azure-sdk-for-c/pull/867), it is an interesting usage scenario, so I updated it based on the new APIs. Please skim, @vhvb1989.